apps/chat/src/components/Chat/Chat.tsx

```diff
@@ -294,6 +301,31 @@ const ChatView = memo(() => {
     }
   }, []);
 
+  const handleChatMessagesResize = useCallback(() => {
+    if (
+      chatMessagesRef.current &&
+      !messageIsStreaming &&
+      mergedMessages.length
+    ) {
+      handleScroll();
+    }
+  }, [handleScroll, mergedMessages.length, messageIsStreaming]);
+
+  const handleChatResize = useCallback(() => {
+    if (
+      !isWideScreen() &&
```

Comment:
`isWideLayout` should be for `isWideScreen()`. Right?
```suggestion
      isWideScreen() &&
```